core/src/main/java/org/owasp/encoder/Encode.java

```diff
@@ -276,6 +276,8 @@ public static void forHtmlContent(Writer out, String input)
      *
      * <p><b>Additional Notes</b></p>
      * <ul>
+     * <li>When using this method, the caller must provide quotes around the attribute value.</li>
```

Comment:
Don't like this phrasing, but I chose it because `forHtmlUnquotedAttribute()` JavaDocs use similar phrasing.